src/fides/api/ops/service/masking/strategy/masking_strategy_aes_encrypt.py

```diff
@@ -40,6 +40,7 @@ def mask(
     ) -> Optional[List[str]]:
```

Comment:
Should we change the `values:` typehint to `Optional[List[Optional[str]]]`?

Reply:
good catch, yes, there can be Nones in this list